environment.yml

```diff
@@ -34,8 +34,8 @@ dependencies:
 - pyyaml~=5.1.2
 - reentry~=1.3
 - simplejson~=3.16
-- sqlalchemy-utils~=0.36.0
-- sqlalchemy~=1.3.10
+- sqlalchemy-utils~=0.37.0
```

Comment:
It appears that version 0.3.7.0 has not been released for conda-forge yet: https://anaconda.org/conda-forge/sqlalchemy-utils

https://github.com/conda-forge/sqlalchemy-utils-feedstock

Reply:
The version has been released on conda-forge in the meantime.